docs/topics/compiler-reference.md

```diff
@@ -218,6 +226,13 @@ into the classpath.
 
 Script definition template classes. Use fully qualified class names and separate them with commas (**,**).
 
+### -Xjvm-expose-boxed
+
+<primary-label ref="experimental-general"/>
+
+Generate boxed versions of all inline value classes in the module, making them accessible from Java. For more information, see [Inline value classes](java-to-kotlin-interop.md#inline-value-classes)
```

Comment:
This is not entirely correct. This generated boxed versions of all inline value classes, but also versions of the functions using those boxed inline value classes that can be consumed from Java.